package.json

```diff
@@ -9,6 +9,7 @@
 		"storybook": "pnpm run -C site/ storybook"
 	},
 	"devDependencies": {
+		"@biomejs/biome": "1.9.4",
```

Comment:
no

Reply:
biome extension doesn't work in vscode unless you have dep in root dir :cry: